MoTrPAC/motrpac-frontend
# Connects to #86. Methods Page UI Updates.

### Key Changes

1. Moved the _study docs_ table from the _Releases_ page to the _Methods_ page.
2. Enabled the _Methods_ nav item on the sidebar.
3. Renamed label instances of "Reference Standards" to "QC-Reference" on _Dashboard_ page.
4. Updated/fixed test assertions.
5. Added/updated Storybook components.

#### Steps to test

1. Sign into your local instance of this branch.
2. On the *Dashboard* page, expect to see previously named "Reference Standards" labels having been changed to "QC-Reference".
3. On the *Methods* page, expect to see the _study docs_ table with an external link to the right.
4. On the *Releases* page, expect to see the _study docs_ table *no longer* present to either _internal_ or _external_ releases.
5. When running the `yarn test` command-line in the local working copy of this branch, expect to see all test passed.
